spinn_front_end_common/utility_models/data_speed_up_packet_gatherer_machine_vertex.py

```diff
@@ -272,22 +279,57 @@ def get_local_provenance_data(self):
                                     n_lost_seq_nums))))
         return prov_items
 
-    @staticmethod
     def set_cores_for_data_extraction(
-            transceiver, extra_monitor_cores_for_router_timeout,
+            self, transceiver, extra_monitor_cores_for_router_timeout,
```

Comment:
Can't we call this parameter `extra_monitor_cores`? The extra length doesn't add very much information yet it forces a lot more backslashes in the code.

Also, this is a public method without any documentation comment.

Reply:
Happy enough to make these changes, but these aren't changes made here (the problem you mention is already in master).